test/gst-msdk/encode/hevc.py

```diff
@@ -89,6 +90,37 @@ def test_r2r(self, case, gop, slices, qp, quality, profile):
     vars(self).setdefault("r2r", 5)
     self.encode()
 
+class cqp_lp_444(HEVC8EncoderTest):
```

Comment:
Why do we need a new class for this?  It is the same as the `cqp_lp` class, just different spec and default profiles.  We should use the `cqp_lp` spec and class, instead, for 444 cases.  The profile can be specified in user spec explicitly.

Reply:
I had wanted to separate the hevc 8bit 444 encode from hevc 8bit encode test result in the test report in future. Just as:
Encode:  hevc
               hevc.lp
               hevc.lp.444
What do you have any suggestion on separate test result in test report?

Reply:
You can filter by profile parameter for now.

In the future, I'd like to add support for user-defined categories that allow users to assign cases to arbitrary categories that can be used to filter sets of cases (e.g. "itu-t", "4k", "allegro", "multires", or any arbitrary user-defined category, etc...).  But we have not designed that feature yet.

Reply:
Ok, I followed your suggestion and updated the patches